templates/agent/encryption.cfg.erb

```diff
@@ -0,0 +1,2 @@
+ENCRYPTED=yes
```

Comment:
why did you choose an erb template here?

Reply:
I'll change this in my next PR (which will add some acceptance tests for this parameter)